mozci/push.py

```diff
@@ -294,14 +296,36 @@ def add(result):
         except MissingDataError:
             pass
 
-        # Gather information from the unittest table. We allow missing data for this table because
-        # ActiveData only holds very recent data in it, but we have fallbacks on Taskcluster
-        # artifacts.
-        try:
-            add(run_query("push_tasks_results_from_unittest", args))
-        except MissingDataError:
-            pass
+        # Let's gather error/results from cache or AD/Taskcluster
+        test_tasks_results = adr.config.cache.get(self.push_uuid, {})
+        if len(test_tasks_results.keys()) > 0:
+            tasks = self.normalized_tasks(tasks)
+            # Let's add cached error summaries to TestTasks
+            for t in tasks:
+                if isinstance(t, TestTask):
+                    error_summary = test_tasks_results.get(t.id)
+                    # Only Test tasks have stored error summary information in the cache
+                    if error_summary:
+                        t._errors = error_summary["errors"]
+                        t._results = error_summary["results"]
+            logger.info(
+                "Fetched tasks errors/results for {} from the cache".format(
+                    self.push_uuid
+                )
+            )
```

Comment:
Same here, either remove or make it debug (it's not really information a consumer of the library is interested in).